crates/adapters/src/format/avro/schema.rs

```diff
@@ -114,13 +122,12 @@ fn validate_map_schema(avro_schema: &AvroSchema, value_schema: &ColumnType) -> R
 /// Check that Avro schema can be deserialized as SQL `TIMESTAMP` type.
 fn validate_timestamp_schema(avro_schema: &AvroSchema) -> Result<(), String> {
     // TODO: we can support TimestampMillis by transforming them to micros on the fly.
-    if avro_schema == &AvroSchema::TimestampMillis {
-        return Err("Avro timestamp encoding using 'timestamp-millis' type is currently not supported; use 'timestamp-micros instead'".to_string());
-    }
-
-    if avro_schema != &AvroSchema::TimestampMicros && avro_schema != &AvroSchema::Long {
+    if avro_schema != &AvroSchema::TimestampMicros
```

Comment:
why does the above comment belong in this function?
This name AvroSchema seems wrong, it's not a schema, it's a property of how a particular data type is encoded